projects/aca-content/src/lib/components/header-actions/header-actions.component.scss

```diff
@@ -0,0 +1,55 @@
+.aca-page-layout-header {
+
+  .app-search-input {
+    background: none;
+  }
+
+  .action-bar {
+    display: flex;
+    flex: auto;
+    height: 32px;
+    margin-left: 24px;
+
+    adf-toolbar-divider {
+      width: 24px !important;
+      height: 32px !important;
+      margin: 4px 0 0 12px !important;
+    }
+  }
+
+  .aca-mat-button {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    letter-spacing: 0.25px;
+    font-weight: 500;
+    font-size: 14px;
+    font-style: normal;
+    border-radius: 6px;
+    border: none;
+  }
+
+  .aca-create-button {
+    width: 71px !important;
+    min-width: 71px;
+    height: 32px;
+    background-color: var(--theme-create-button-background-color);
+    color: var(--theme-create-button-text-color);
+    text-overflow: ellipsis;
+    display: flex;
+    padding: 0;
+  }
+
+  .aca-upload-button {
+    width: 73px !important;
```

Comment:
why this is 73px and important? what if it's another language and it needs more space?